service/src/main/java/greencity/service/EventCommentServiceImpl.java

```diff
@@ -227,7 +229,7 @@ public void delete(Long eventCommentId, UserVO user) {
         if (eventComment.getComments() != null) {
             eventComment.getComments().forEach(comment -> comment.setDeleted(true));
```

Comment:
should we set EventCommentStatus.DELETED for every comment here too?

Reply:
fixed